### DISCUSSION TITLE
Two esp32 - internal install mod

### DISCUSSION THREAD

**Discussion by itslemcy** [darthcloud/BlueRetro#309] [Q&A]
Hi, so i was thinking of installing internally in my atari jaguar two esp32 to have a full wireless experience.

When i turn on the console, how can i know that the first controller will pair with the esp32 on controller port 1 and not 2? Is there a workaround to guarantee the correct connection/pairing order?

Question n2: if i connect a jaguar OG controller to that port, what would happen?

**Comment by darthcloud** [comment]
You would need to set inquiry mode to manual in the advance config for both.

And using the boot button set it manualy on on one and pair the controller, then do same with second one.

Once paired they will go automatically on the good one.
